QEfficient/utils/__init__.py

```diff
@@ -13,3 +16,27 @@
     onnx_exists,
     qpc_exists,
 )
+from QEfficient.utils.logging_utils import logger
+
+
+def padding_check_and_fix(
+    tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast]
+):
+    """
+    Checks Tokenizer paddding side and pad_token_id viability.
+    --------
+    
+    Tokeinizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast]. Pass model tokenizer to check and fix.
+    """
+    
+    if tokenizer.padding_side != "left":
```

Comment:
can you align with new interface directly?

Reply:
Hey Anuj, for new interface single change is to change this to `right`, don't want to wait for that PR, we will update this in that PR anyway after rebase.